src/parser/parse_tree_cl.c

```diff
@@ -4397,6 +4397,10 @@ pt_select_list_to_one_col (PARSER_CONTEXT * parser, PT_NODE * node, bool do_one)
 	    {
 	      do_rewrite = true;	/* give up */
 	    }
+	  else if (node->info.query.order_by && node->info.query.limit)
+	    {
+	      do_rewrite = true;
+	    }
```

Comment:
**do_rewrite** should be initialized because at the next "else" line do_rewirte is checked.